src/event.rs

```diff
@@ -444,6 +444,7 @@ impl Command for DisableBracketedPaste {
 /// * [kitty terminal](https://sw.kovidgoyal.net/kitty/)
 /// * [foot terminal](https://codeberg.org/dnkl/foot/issues/319)
 /// * [WezTerm terminal](https://wezfurlong.org/wezterm/config/lua/config/enable_kitty_keyboard.html)
+/// * [alacritty terminal](https://github.com/alacritty/alacritty/commit/cb03806e2ab85674c45e87e1bb24dfe2fd1a918c)
```

Comment:
What about linking to the PR instead?

```suggestion
/// * [alacritty terminal](https://github.com/alacritty/alacritty/pull/7125)
```
Or the issue
```suggestion
/// * [alacritty terminal](https://github.com/alacritty/alacritty/issues/6378)
```